fedimg/services/ec2/ec2imgpublisher.py

```diff
@@ -192,6 +192,22 @@ def publish_images(self, region_image_mapping=None):
                     )
                 )
 
+                fedimg.messenger.notify(
+                    topic='image.upload',
```

Comment:
what's the difference between `image.upload` and `image.publish`?

Reply:
None, this part was added in order for get_ami.py to work. I'll update the get_ami script and deprecate this part of code.